seekh-sdk/src/auth/index.ts

```diff
@@ -0,0 +1,32 @@
+import axios from 'axios';
+import { login, signup } from './types';
+
+const api = axios.create({ baseURL: 'http://localhost:5000' })
+
+class Auth {
+    static login(loginCredentials: login) {
+        return new Promise((resolve, reject) => {
```

Comment:
Is there another way to return promises? We are not taking of advantage of async await here.